src/app/components/feed/SaveFeed.js

```diff
@@ -486,11 +486,11 @@ const SaveFeed = (props) => {
           />
           { feed.id ?
             <FeedActions
+              feedTeam={{ ...feedTeam, permissions: feedTeam.permissions }}
```

Comment:
Is this necessary? Isn't permissions already part of the `feedTeam` prop/object?

Reply:
I think this was to make the linter happy, otherwise it would say that the field was requested but not used.

Reply:
I see that it was added to `fragment FeedActions_feedTeam on FeedTeam` so it shouldn't be necessary here